EventsExpress.Core/Services/InventoryService.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.Infrastructure;
+using EventsExpress.Core.IServices;
+using EventsExpress.Db.Entities;
+using EventsExpress.Db.IRepo;
+
+namespace EventsExpress.Core.Services
+{
+    public class InventoryService : IInventoryService
+    {
+        private readonly IUnitOfWork _db;
+        private readonly IMapper _mapper;
+
+        public InventoryService(
+            IUnitOfWork unitOfWork,
+            IMapper mapper)
+        {
+            _db = unitOfWork;
+            _mapper = mapper;
+        }
+
+        public async Task<OperationResult> AddInventar(Guid eventId, InventoryDTO inventoryDTO)
+        {
+            var ev = _db.EventRepository.Get("Inventories").FirstOrDefault(e => e.Id == eventId);
```

Comment:
If you are checking for existance of event - consider not joining inventory here.

Reply:
Solved